.github/workflows/ci.yml

```diff
@@ -105,8 +105,29 @@ jobs:
             - name: Run tests
               shell: bash -l {0}  # required to activate the conda environment
               env: ${{ fromJSON(needs.build.outputs.images) }}
-              run: |
-                  pytest -v
+              run: pytest -v -m "not integration"
```

Comment:
How is this `-m` option work?

Reply:
https://docs.pytest.org/en/7.1.x/example/markers.html#marking-test-functions-and-selecting-them-for-a-run

(I find the not operator quite strange, but it works)

Reply:
never mind. I find it, a useful option.